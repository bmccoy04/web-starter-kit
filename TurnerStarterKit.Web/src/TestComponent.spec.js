import React from 'react';
import {create} from 'react-test-renderer';

const TestComponent = ({test}) => <div>{test}</div>;

describe('test component', () => {
  it('sample test', () => {
    const tree = create(<TestComponent test="foo" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
