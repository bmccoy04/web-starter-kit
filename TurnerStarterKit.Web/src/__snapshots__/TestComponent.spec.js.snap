exports[`test component sample test 1`] = `
<div>
  foo
</div>
`;
